src/test/java/org/apache/tinkerpop/gremlin/tinkergraph/storage/SerializerTestVertex.java

```diff
@@ -0,0 +1,144 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.tinkerpop.gremlin.tinkergraph.storage;
+
+import org.apache.tinkerpop.gremlin.structure.VertexProperty;
+import org.apache.tinkerpop.gremlin.tinkergraph.structure.*;
+import org.apache.tinkerpop.gremlin.tinkergraph.structure.specialized.gratefuldead.FollowedBy;
+import org.apache.tinkerpop.gremlin.tinkergraph.structure.specialized.gratefuldead.SungBy;
+import org.apache.tinkerpop.gremlin.tinkergraph.structure.specialized.gratefuldead.WrittenBy;
+import org.apache.tinkerpop.gremlin.util.iterator.IteratorUtils;
+
+import java.io.Serializable;
+import java.util.*;
+
+public class SerializerTestVertex extends SpecializedTinkerVertex implements Serializable {
+    public static final String label = "testVertex";
```

Comment:
same as in the other test, I don't think those need to be public

Reply:
it's just a test class. also note that these are (and should be) public for other specialized vertices, so that we can reference the label e.g. `graph.V.hasLabel(Foo.label)`